ex7.4_NumPy_matrix_zero_one.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def create_matrix_zero_one():
+    """
+  function that create a 10x10 matrix, in which the elements on the borders will be equal to 1, and inside 0
+    :return: 10x10 matrix with one's outside and inside zeros.
+    """
+    matrix = np.ones((10, 10))
+    matrix[1:-1, 1:-1] = 0
+    return matrix
+
+
+if __name__ == '__main__':
+    print(create_matrix_zero_one())
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline